src/pages/scientificServices/pipelines/views/JobHistory.tsx

```diff
@@ -138,15 +138,15 @@ const getColumns = (paginatedRuns: PipelineRun[]) => {
       cellRenderer: ({ rowIndex }) => {
         return <DescriptionCell pipelineRun={paginatedRuns[rowIndex]} />;
       },
-      size: { basis: 140 },
+      size: { basis: 150 },
```

Comment:
just resizing some of the columns to look a little bit better on smaller screens